### PR TITLE
Docker and docker-compose setup

### PR DESCRIPTION
**Pull request by batovsky**
Hey,

had some fun with compose setup, was handy for testing.

### CODE REVIEW

**Comment by batovsky** [comment]
Thanks for quick reply and congratulations, I am ready to help, kindly ask you for review.

File: promeditor/README.md [hunk 1]
```diff
@@ -42,6 +42,22 @@ If your Prometheus is running on a different host, you need to adjust the `proxy
 
 3. Look at `src/App.js` for an example on how to include the editor field into your app.
 
+### 4. Using Docker
```
Comment (davkal) [inline promeditor/README.md]: This should be 3.

File: promeditor/README.md [hunk 2]
```diff
@@ -42,6 +42,22 @@ If your Prometheus is running on a different host, you need to adjust the `proxy
...
+
+Adjust Prometheus URL in [package.json](package.json) for your docker setup.
```
Comment (davkal) [inline promeditor/README.md]: An example might help here.

**Comment by davkal** [comment]
So close! Just needs a fixup for the readme.

**Comment by batovsky** [comment]
heh, and another round, please :-)

**Comment by davkal** [comment]
Excellent, thank you for the contribution @batovsky !

**Pull request merged**